general/internal-user/template.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: denyexternalusers
+spec:
+  crd:
+    spec:
+      names:
+        kind: DenyExternalUsers
+      validation:
+        # Schema for the `parameters` field
+        openAPIV3Schema:
+          properties:
+            labels:
+              type: array
+              items: 
+                type: string
+            employeeDomains:
+              type: array
+              items: 
+                type: string
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package denyexternalusers
+
+        violation[{"msg": msg}] {
+          obj := input.review.object
+          obj.kind == "RoleBinding"
+          email := obj.subjects[0].name
+          not isEmployee(email)
```

Comment:
As mentioned in the other comment

internal = isEmail && not isEmployee

We just need to add the isEmail logic here